tests/test_api_example.py

```diff
@@ -141,8 +142,8 @@ def test_api_example(test_api_obj):
 
     except Exception as e:
         print(e)
-        if conf.api_url == 'http://127.0.0.1:5000':
-            test_api_obj.write("Please run the test against http://35.167.62.251/ by changing the api_url in api_example_conf.py")
+        if base_url_conf.api_base_url == 'http://127.0.0.1:5000':
+            test_api_obj.write("Please run the test against http://35.167.62.251/ by changing the api_base_url in base_url_conf.py")
```

Comment:
`http://35.167.62.251/` seems to throw a `401`, can you check if we have a domain associated with our hosted `cars-api` service and use it here, instead of the `IP`